Frameworks/UIKit/UIBezierPath.mm

```diff
@@ -316,97 +277,72 @@ - (void)stroke {
 /**
  @Status Interoperable
 */
-- (void)dealloc {
-    if (_pattern) {
-        free(_pattern);
-        _pattern = NULL;
-    }
-
-    CGPathRelease(_path);
-    [super dealloc];
-}
-
-/**
- @Status Interoperable
-*/
-- (void)setLineWidth:(CGFloat)width {
-    _lineWidth = width;
-}
-
-/**
- @Status Interoperable
-*/
-- (CGFloat)lineWidth {
-    return _lineWidth;
-}
+- (void)setLineDash:(const CGFloat*)pattern count:(NSInteger)count phase:(CGFloat)phase {
+    _patternCount = count;
+    _dashPhase = phase;
 
-/**
- @Status Interoperable
-*/
-- (void)setLineJoinStyle:(CGLineJoin)join {
-    _lineJoin = join;
+    if (count > 0) {
+        std::vector<CGFloat> newPattern(pattern, pattern + count);
+        _pattern = std::move(newPattern);
+    }
 }
 
 /**
  @Status Interoperable
 */
-- (void)setUsesEvenOddFillRule:(BOOL)usesEvenOdd {
-    _usesEvenOddFillRule = usesEvenOdd;
+- (void)removeAllPoints {
+    _workingPath = woc::MakeStrongCF(CGPathCreateMutable());
 }
 
 /**
  @Status Interoperable
 */
-- (void)setLineDash:(const CGFloat*)pattern count:(NSInteger)count phase:(CGFloat)phase {
-    if (_pattern) {
-        free(_pattern);
-        _pattern = NULL;
+- (void)getLineDash:(CGFloat*)pattern count:(NSInteger*)count phase:(CGFloat*)phase {
+    if (phase) {
+        *phase = _dashPhase;
+    }
+    if (count) {
+        *count = _patternCount;
     }
 
-    _patternCount = count;
-    _dashPhase = phase;
-
-    assert(sizeof(float) == sizeof(CGFloat));
-    if (count > 0) {
-        _pattern = (float*)malloc(sizeof(float) * count);
-        memcpy(_pattern, pattern, sizeof(float) * count);
+    if (pattern && _patternCount > 0) {
+        std::copy(_pattern.begin(), _pattern.end(), pattern);
     }
 }
 
 /**
  @Status Stub
 */
-- (void)removeAllPoints {
+- (void)fillWithBlendMode:(CGBlendMode)blendMode alpha:(CGFloat)alpha {
     UNIMPLEMENTED();
```

Comment:
technically stupidly easy to implement... and the others should be implemented _in terms of these ones_ by passing kCGBlendNormal or kCGBlendSourceOver